packages/backend/src/template/template.dto.ts

```
@@ -0,0 +1,87 @@
+  Matches,
+} from 'class-validator';
+import { VALID_IMAGE_ID_PATTERN } from '../image/image.constants';
+
+/**
+ * A listing's audience — see docs/design/template-gallery.md.
+ *
+ * `public` is accepted by the model but refused by the service until the
+ * Phase 3 review pipeline exists: listing content that nothing reviews is the
+ * one failure mode a template gallery cannot walk back.
+ */
+export const TEMPLATE_VISIBILITIES = ['unlisted', 'workspace', 'public'] as const;
```

ESLint

[error] Replace `'unlisted',·'workspace',·'public'` with `⏎··'unlisted',⏎··'workspace',⏎··'public',⏎`

(prettier/prettier)